.github/workflows/pull_request.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        go-version: [1.18, 1.19]
+        go-version: [1.19, 1.20]
```

Comment:
From https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html

YAML converts certain strings into floating-point values, such as the string 1.0. If you need to specify a version number (in a requirements.yml file, for example), you will need to quote the value if it looks like a floating-point value:

version: "1.0"

Can we put 1.19 and 1.20 in quotes please